### DISCUSSION TITLE
Fade in/out effect without re-encoding the whole video?

### DISCUSSION THREAD

**Discussion by wbadlands** [mifi/lossless-cut#1610] [Ideas]
Hello. I think this was requested before by a couple people.

Is it even possible to implement cutting / joining with fade in / out effect the way I'm thinking? As in, only re-encode the necessary frames (the few seconds that it fades in and out), while keeping the rest of the video lossless? I'm thinking it's possible since it already does something similar with the Precise Cutting option, if I'm not mistaken.

It would be a nice addition. Love the program, keep it up!

**Comment by mifi** [comment]
hi. that might be possible, but it's not super trivial. If you mean fade one clip into another clip, then I think it will go into #976

**Reply by wbadlands** [reply]
Yeah, like clip 1 fades out into a black screen, and clip 2 fades in from a black screen. And not just when you drag and drop multiple clips and concatenate them, but also when editing a large video and creating many segments from it, allow each segment to fade out / in into each other. Maybe have a checkbox that would be like "Use Fade Effect For Every Segment", and have 2 boxes that we can enter the fade in and fade out duration times. This way we wouldn't need to set the fade effect for each individual segment.

Don't think there's any tool out there that can do that without having to re-encode the whole video.

Thank you!

**Comment by mifi** [comment]
added it here: #372 - also related: #1044
